notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettSakLoeser.kt

```diff
@@ -70,6 +77,24 @@ class OpprettSakLoeser(
     }
 
     override fun onBehov(behov: Behov) {
+        val utloesendeMelding = behov.jsonMessage.toJson().parseJson()
+        val transaksjonId = utloesendeMelding.toMap()[Key.UUID]
+            ?.fromJson(UuidSerializer)
```

Comment:
hva med: 
`        val transaksjonId = utloesendeMelding.toMap()[Key.UUID]
            ?.fromJson(UuidSerializer)
            ?: UUID.randomUUID()
            .also {
                sikkerLogger.error(
                    "Mangler transaksjonId i ${simpleName()}. Erstatter med ny, tilfeldig UUID '$it'." +
                        "\n${utloesendeMelding.toPretty()}"
                )
            }`

Reply:
Se https://github.com/navikt/helsearbeidsgiver-inntektsmelding/pull/326#discussion_r1420046392